source/run-command/tests/unified/run-command.yml

```diff
@@ -0,0 +1,242 @@
+description: runCommand spec requirements
+
+schemaVersion: "1.5"
+
+createEntities:
+  - client:
+      id: client
+      observeEvents: [commandStartedEvent]
+  - database:
+      id: plainDb
+      client: client
+      databaseName: plainDb
+  - collection:
+      id: collection
+      database: plainDb
+      collectionName: collection
+  - database:
+      id: dbWithRC
+      client: client
+      databaseName: dbWithRC
+      databaseOptions:
+        readConcern:
+          level: 'local'
+  - database:
+      id: dbWithWC
+      client: client
+      databaseName: dbWithWC
+      databaseOptions:
+        writeConcern:
+          w: 0
+  - session:
+      id: session
+      client: client
+  # Stable API test
+  - client:
+      id: stableApiClient
+      observeEvents: [commandStartedEvent]
+      serverApi:
+        version: "1"
+        strict: true
+  - database:
+      id: stableApiDb
+      client: stableApiClient
+      databaseName: stableApiDb
+
+initialData:
+- collectionName: collection
+  databaseName: plainDb
+  documents: [ { _id: 0 } ]
+
+tests:
+  - description: always attaches $db to given command
+    operations:
+      - name: runCommand
+        object: plainDb
+        arguments:
+          commandName: ping
+          command:
+            ping: 1
+        expectResult:
+          ok: 1
+    expectEvents:
+      - client: client
+        events:
+        - commandStartedEvent:
+              command:
+                ping: 1
+                $db: plainDb
+              commandName: ping
+
+  - description: attaches the provided session lsid to ping command
```

Comment:
you say "given command" in the other test cases

Reply:
fixed